views/tour.dt

```diff
@@ -9,6 +9,7 @@ block head
 	link(rel="stylesheet", href="/static/css/ansi.css")
 
 block content
+	script var translations = #{translations.toJson};
```

Comment:
I'm not a huge fan of this. I guess ideally we would have a separate route which returns a json file per language. The returned translation file could be nicely cached.
However, 
- currently it's not a huge overhead
- I plan on avoiding the page reloads and making them as pure Ajax requests instead -> the translation file would only be requested once.